pyartcd/pyartcd/jenkins.py

```diff
@@ -279,6 +280,20 @@ def start_build_sync(build_version: str, assembly: str, doozer_data_path: Option
     )
 
 
+def start_cincinnati_prs(from_releases: list, release_name: str, advisory_id: int,
+                         candidate_pr_note: str, skip_ota_notification, **kwargs) -> Optional[str]:
+    return start_build(
+        job=Jobs.CINCINNATI_PRS,
+        params={
+            'FROM_RELEASE_TAG': ','.join(from_releases),
+            'RELEASE_NAME': release_name,
+            'ADVISORY_NUM': advisory_id,
+            'CANDIDATE_PR_NOTE': candidate_pr_note,
+            'SKIP_OTA_SLACK_NOTIFICATION': skip_ota_notification
```

Comment:
Just wanted to note that the job that's being called has a `GITHUB_ORG` param with a default value of `openshift`. Since joep mentioned that the repos will soon be migrated, it might be a good idea to explicitly pass that so that we can keep track of this? Just a nit though

Reply:
Added :+1: